overview/load.adoc

```diff
@@ -0,0 +1,34 @@
+include::../partials/attributes.adoc[]
+//Category: skupper-overview
+// Type: assembly
+[id="skupper-load"] 
+= Skupper load balancing and failover
+
+Skupper enables load balancing and failover across servers located across the service network.
+Specifically, Skupper balances *active TCP connections* across workloads deployed in distinct sites.
+If a workload at one site becomes unavailable, traffic is automatically rerouted to available sites. 
+For example, if you deploy the same backend code on two sites and expose the backend on the service network, concurrent requests from a third site to the backend service are processed by both sites.
+
+In a service network, the routing algorithm attempts to use the path with the lowest total *cost* from client to target server.
```

Comment:
The algorithm attempts to use the target server with the lowest number of open connections, offset by the link cost.  In other words, the backlog (open connections) plus the link cost is used to select the outgoing path to the targets.

Reply:
@ted-ross I get what you're saying but trying to keep it simple in this section..

WDYT of the following (which would introduce a new term, but would allow us say 'the routing algothithm attempts to use the path with the lowest operational cost'):

Operational cost = ∑(Open connections) + ∑(Link costs)

Reply:
I wasn't necessarily offering alternative text.  I just wanted to point out the dynamic nature of the balancing algorithm.  Simply choosing the lowest cost is a very static approach.  We use cost to strengthen the 
"locality" of traffic (as you point out below).